Online/State/RealizedPlayerState.cs

```diff
@@ -35,6 +35,16 @@ public class RealizedPlayerState : RealizedCreatureState
         public float tongueRequestedLength;
         [OnlineField(group = "tongue", nullable = true)]
         public BodyChunkRef tongueAttachedChunk;
+        [OnlineField]
+        public bool reachingForObject;
+        [OnlineField]
+        public Vector2 absoluteHuntPos;
+        [OnlineField]
+        public Vector2 tailPos3;
+        [OnlineField]
+        public Vector2 tailPos2;
```

Comment:
this is a LOT of floats, totals 24 bytes.

Could we maybe use `[OnlineFieldHalf]` so they're half-encoded?

Halfs have ~1 increments in 1k~2k range so for rooms bigger than that that could be a problem (that one shoreline room is 6k pixels wide, on the far right side you'd have an error of up to 4px on the half-encoded positions. A way around that is to encode these relative to mainbodychunk.pos instead so they're operating on a small range.

Reply:
We can actually just close this whole PR. I'm going to remove the half baked implementation at the moment because there's more important items to work on.